Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
```

Comment:
>Unreleased [](http://example.com/codeflow?start=3&length=10)

You updated the SDK but there's nothing in the release notes about it. Did we need to do that in order to accept a bug fix relevant to this tool? If so, we should mention that. If not, it's fine not to mention the SDK update).

Remember, 'was there a potentially customer visible impact to the change?'. If so, talk about it. If not, ok, silence is golden. :)